source/angelscript_addons/scriptbuilder/scriptbuilder.h

```diff
@@ -107,15 +101,15 @@ class CScriptBuilder
 	// Get metadata declared for class variables
 	const char *GetMetadataStringForTypeProperty(int typeId, int varIdx);
 
-	// Get metadata declared for class functions
+	// Get metadata declared for class methods
 	const char *GetMetadataStringForTypeMethod(int typeId, asIScriptFunction *method);
 #endif
 
 protected:
 	void ClearAll();
 	int  Build();
 	int  ProcessScriptSection(const char *script, unsigned int length, const char *sectionname, int lineOffset);
-	virtual int  LoadScriptSection(const char *filename); // ## Rigs of Rods modification
+	int  LoadScriptSection(const char *filename);

```

Comment:
You broke the thing, RoR will be unable to load scripts. :slightly_frowning_face: The `virtual` hack was there for a reason. Without it, we'll end up calling the base implementation which only searches local paths, but we need to call our overload which uses OGRE resource system.